daffodil-core/src/main/scala/org/apache/daffodil/dsom/ElementBase.scala

```diff
@@ -701,6 +704,7 @@ trait ElementBase
                   alignInBits, implicitAlignmentInBits, primType.name, this.knownEncodingName)
             }
             case Representation.Binary => primType match {
+              case PrimType.HexBinary => 8
```

Comment:
I think the 8 here does nothing? This part of this val is just doing validation like making sure packed decimals are aligned. We don't have that restriction with hex binary since we allow hexbinary to be non-byte sized and non-byte aligned. So I think this can just be a no-op like the Float/Double/Booleans below it?